cli_reference/basic_cli_operations.adoc

```diff
@@ -136,32 +190,70 @@ One of the fundamental capabilities of OpenShift is the ability to build applica
 
-The syntax to describe a deployment configuration in human-readable form is:
+To describe a deployment configuration in human-readable form:
 
 ****
-`osc deploymentConfigs describe [replaceable]#<deployment config>#`
+`$ osc describe dc _<deployment_config>_`
 ****
 
-=== Deployment Rollbacks
+For example:
 
-Rollbacks revert an application back to a previous deployment. Note that environment variables and volumes are included in rollbacks, so there are a variety of considerations to make when deciding whether a rollback is viable, such as:
+====
+
+[options="nowrap"]
+----
+$ osc describe dc docker-registry
+Name:		docker-registry
+Created:	18 hours ago
+Labels:		docker-registry=default
+Latest Version:	1
+Triggers:	Config
+Strategy:	Recreate
+Template:
+	Selector:	docker-registry=default
+	Replicas:	1
+	Containers:
+		NAME		IMAGE					ENV
+		registry	openshift/origin-docker-registry:v0.4.3	OPENSHIFT_CA_DATA=[omitted for space],OPENSHIFT_MASTER=https://10.245.2.2:8443
+Latest Deployment:
+	Name:		docker-registry-1
+	Status:		Complete
+	Selector:	deployment=docker-registry-1,deploymentconfig=docker-registry,docker-registry=default
+	Labels:		docker-registry=default
+	Replicas:	1 current / 1 desired
+	Pods Status:	1 Running / 0 Waiting / 0 Succeeded / 0 Failed
+----
+====
 
-* If security credentials have been recently updated, the previous deployment may not have the correct values.
-* If the previous deployment used a custom strategy, which is no longer available or usable, the deployment may not be deployed correctly.
+*Deployment Rollbacks*
 
-By default, only the pod and container configuration will be changed and scaling or trigger settings will be left as-is.
+Rollbacks revert an application back to a previous deployment. Note that
+environment variables and volumes are included in rollbacks, so there are a
+variety of considerations to make when deciding whether a rollback is viable,
+such as:
 
-Using the `-d`, or `--dry-run` option prints a human-readable representation of the updated deployment configuration instead of executing the rollback. This is useful for inspecting a desired outcome before an actual rollback.
+- If security credentials have been recently updated, the previous deployment
+may not have the correct values.
+- If the previous deployment used a custom strategy which is no longer available
+or usable, the deployment may not be deployed correctly.
 
-Use the `osc rollback` command to revert part of an application back to a previous deployment. The syntax for `rollback` is:
+By default, only the pod and container configuration are changed and scaling or
+trigger settings are left as-is.
+
+Using the `-d`, or `--dry-run` option prints a human-readable representation of
+the updated deployment configuration instead of executing the rollback. This is
+useful for inspecting a desired outcome before an actual rollback.
+
+Use the `osc rollback` command to revert part of an application back to a
+previous deployment:
 
```

Comment:
[Suggestion]
The `-d`, or `--dry run` option shows the configuration of the updated deployment in an easy to read format without actually executing the rollback. This allows you to inspect the output before actually proceeding with the rollback.